### ISSUE TITLE
URL for P2 not loading

### ISSUE THREAD

**Issue by janenycxo** [susanBuck/e2-fall19#34]
I have been trying to get my p2 domain up to no avail. It still shows hello from my new digitalocean server although i went through the steps to root ip address several times. i did receive an error upon gitpull and had to go back and merge some files but that resolved the gitpull error. 

http://p1.janenycxo.me is working as it should
http://p2.janenycxo.me does not populate the game page.

I would appreciate it if anyone can direct me to what i am missing? TIA!

**Comment by RGPhD** [comment]
Dear @janenycxo,

The only thing I can think of is if you might have deployed the updated files to your digital ocean **/var/www/html** directory instead of your **/var/www/html/e2** directory.  You mentioned you received an error and then you had to merge files; I also received that message and then after I merged the files I was able to git pull and git push to Github and then git pull to digital ocean.

I am a beginner, so I went back and followed each step that Professor Buck went over in the Week 1 Part 8 video at <https://www.youtube.com/watch?v=Sseblb9SRdk>.

Wish I could be of more help; good luck.
Robelyn

**Comment by janenycxo** [comment]
HI @RGPhD ,
I am going to screenshot my steps, maybe it will be easier to see what i missed that way as I am doing as the notes show me and referencing the prof's video tutorial. I am sure I'll get it...just wished I could see it.
[URL Not Deploying.docx](https://github.com/susanBuck/e2-fall19/files/3678684/URL.Not.Deploying.docx)

Jane

**Comment by RGPhD** [comment]
Hello @janenycxo,

I noticed you have a ```c:xamppapacheconfextrahttpd-vhosts.conf``` file in your GitHub e2 repository.  I do not have that file on GitHub nor have I seen that on our classmates e2 repositories.  I do have the file configued, but it does not show up on GitHub. Your file also differs slightly from the VirtualBlock instructions at <https://hesweb.dev/e2/notes#/production-server/production-domains>.

I am not sure if that matters, I was just looking over your files to see if I could find a bug.

Robelyn

**Comment by janenycxo** [comment]
Hi @RGPhD 

yes, that file popped up preventing me to gitpull. The virtual block is configured the same as the class notes, you must mean the local domain virtual host. I entered the information found in the instructions here https://hesweb.dev/e2/notes#/production-server/summary  

i appreciate your help, i am looking over everything to find a bug but im seeing double vision now....

The last two pages in the document refer to the virtual block
[URL Not Deploying.docx](https://github.com/susanBuck/e2-fall19/files/3678740/URL.Not.Deploying.docx)
Jane

**Comment by janenycxo** [comment]
It might be a digitalocean issue....the page says its not receiving metric data.
[URL Not Deploying.docx](https://github.com/susanBuck/e2-fall19/files/3678980/URL.Not.Deploying.docx)

**Comment by janenycxo** [comment]
I have worked on this all day, retracing each step, but something is not aligning for me. I hope to be able to load the page before our deadline!

**Comment by janenycxo** [comment]
i got it to work!! php is a beast!

**Comment by anyav77** [comment]
Could you explain what was causing the issue and how it got fixed?

**Comment by janenycxo** [comment]
Hi @anyav77,

The php was preventing the p2 from loading. I focused so much attention on committing, pushing and pulling because that is where I felt had the most room for error. I created pages and tested the script separately and they worked fine. However, combined it prevented p2 from loading. As professor explained in lecture videos to practice conservative coding (line by line) I did so in snippets. Finally when I exhausted all possibilities of error and  I really could not “fix” the error I deleted the entire project and worked from scratch. I then compared the new page that deployed against the old page I discovered php codes are very sensitive and was the reason for the page not to load. Removing the call php from first line allowed p2 to load but the game did not function. After several trial and errors I saw that my php game functions also did not align. So all in all, the error was in my coding. I am so relieved and at peace it finally worked for me. Lesson learned...be consistent and code on one page and preview as you go. Thanks to RGPhD for her support!! It was reassuring and helped me get through the project!
-Janenycxo
